Makefile

```diff
@@ -19,7 +19,7 @@ DOCKER_PUBLIC_IMAGE := docker.io/logdna/logdna-agent
 DOCKER_IBM_IMAGE := icr.io/ext/logdna-agent
 
 RUST_COMMAND := $(DOCKER_DISPATCH) $(RUST_IMAGE)
-HADOLINT_COMMAND := $(DOCKER_DISPATCH) $(HADOLINT_IMAGE)
+HADOLINT_COMMAND := $(DOCKER_DISPATCH) $(HADOLINT_IMAGE) ""
```

Comment:
This change seems a little mysterious

Reply:
Yeh, it's a bit awkward, I can move it to where we invoke the hadolint command to make it more obvious, it's just the equivalent of passing a null argument, unfortunately bash doesn't have named args.